setup.py

```diff
@@ -1,9 +1,12 @@
-import sys
+#!/usr/bin/env python
 from setuptools import setup
 
+with open("README", 'r') as f:
```

Comment:
Won't this fail since you renamed the file to README.md?

Reply:
Done